cmd/server/command/start.go

```diff
@@ -456,3 +434,14 @@ func getBroker(c *brokerOptions) (broker.Broker, error) {
 		return nil, errors.New("no broker selected")
 	}
 }
+
+func mapGrafanaOptionsToEnvironment(opts *grafanaOptions) map[string]grafana.Environment {
```

Comment:
Consider moving this function into the file with the other grafanaOptions

Reply:
👍